modules/installation-approve-csrs.adoc

```diff
@@ -129,12 +129,35 @@ $ oc adm certificate approve <csr_name> <1>
 ----
 $ oc get csr -o go-template='{{range .items}}{{if not .status}}{{.metadata.name}}{{"\n"}}{{end}}{{end}}' | xargs oc adm certificate approve
 ----
-
++
 [NOTE]
 ====
 Some Operators might not become available until some CSRs are approved.
 ====
 
+. When all client and server CSRs have been approved, the machines should have the `Ready` status. Verify this by running the following command:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
+[source,terminal]
+----
+NAME      STATUS    ROLES   AGE  VERSION
+master-0  Ready     master  73m  v1.20.0
+master-1  Ready     master  73m  v1.20.0
+master-2  Ready     master  74m  v1.20.0
+worker-0  Ready     worker  11m  v1.20.0
+worker-1  Ready     worker  11m  v1.20.0
+----
++
+[NOTE]
+====
+It could take a few minutes after approval of the server CSRs for the machines to transition to the `Ready` status.
```

Comment:
ISG suggestion to change "could" to "can" or "might". s/It could take a few minutes / It can/might take a few minutes/